cassandra/scripts/init.cql

```diff
@@ -190,6 +189,15 @@ CREATE TABLE IF NOT EXISTS wholesale_supplier.order_line_by_warehouse_item (
 	PRIMARY KEY ((OL_I_ID), OL_W_ID, OL_D_ID, OL_C_ID, OL_O_ID, OL_NUMBER)
 );
 
+-- For T6
+CREATE TABLE IF NOT EXISTS wholesale_supplier.order_line_by_item (
+	OL_I_ID INT,
+	OL_W_ID INT,
+	OL_D_ID INT,
+	OL_O_ID INT,
+	PRIMARY KEY ((OL_I_ID), OL_W_ID, OL_D_ID, OL_O_ID)
+);
```

Comment:
LOL quite similar to the `wholesale_supplier.order_line_by_warehouse_item` but i think we can sort it out later